pkg/test/component_sns_sqs.go

```diff
@@ -0,0 +1,131 @@
+package test
+
+import (
+	"fmt"
+	"github.com/aws/aws-sdk-go/service/sns"
+	"github.com/aws/aws-sdk-go/service/sqs"
+	"log"
+	"net"
+	"strings"
+	"time"
+)
+
+type snsSqsConfig struct {
+	Debug   bool   `mapstructure:"debug"`
+	Host    string `mapstructure:"host"`
+	SnsPort int    `mapstructure:"sns_port"`
+	SqsPort int    `mapstructure:"sqs_port"`
+}
+
+var snsSqsConfigs map[string]*snsSqsConfig
+
+var localstackClients = simpleCache{}
+
+func init() {
+	snsSqsConfigs = map[string]*snsSqsConfig{}
+	localstackClients = simpleCache{}
+}
+
+// do we really need this ?
```

Comment:
i think we should keep it. This ensures that between two tests containers with the same name but different ips get the proper client

Reply:
✅